core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/DetailedStackStatus.java

```diff
@@ -136,6 +136,10 @@ public enum DetailedStackStatus {
     DATABASE_RESTORE_IN_PROGRESS(Status.RESTORE_IN_PROGRESS),
     DATABASE_RESTORE_FINISHED(Status.AVAILABLE),
     DATABASE_RESTORE_FAILED(Status.RESTORE_FAILED),
+    // Determine backup data sizes statuses
+    DETERMINE_BACKUP_DATA_SIZES_IN_PROGRESS(Status.DETERMINE_BACKUP_DATA_SIZES_IN_PROGRESS),
+    DETERMINE_BACKUP_DATA_SIZES_FINISHED(Status.AVAILABLE),
+    DETERMINE_BACKUP_DATA_SIZES_FAILED(Status.AVAILABLE),
```

Comment:
Try to avoid the keyword "BACKUP" in the enums.
With this change, we are just gathering the data size of datalake.